components/automate-cli/pkg/reportgenerator/usagegenerator.go

```diff
@@ -32,34 +34,43 @@ type ConvergeInfo struct {
 
 type ComplianceInfo interface{}
 
-var url = "http://%s:%s"
+var url = "https://%s:%s"
 var errorcsv = "Error while writing csv: "
 var timeFormat = "2006-01-02"
 var dateFormat = "yyyy-MM-dd"
```

Comment:
we are not using the variables dateFormat, datetimeFormat, datetimesecFormat anymore. Can we remove these?

Reply:
Sure.